@radix-ui peer dependency issues with react 19

Hi,

When I try to install this with React 19 via `npm install react@19 react-dom@19 @excalidraw/excalidraw@0.18`

I get some peer dependency warning like below:
```
npm install react@19 react-dom@19 @excalidraw/excalidraw@0.18
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: ex@1.0.0
npm warn Found: react@19.0.0
npm warn node_modules/react
npm warn   peer react@">=16.8.0" from @floating-ui/react-dom@2.1.2
npm warn   node_modules/@floating-ui/react-dom
npm warn     @floating-ui/react-dom@"^2.0.0" from @radix-ui/react-popper@1.2.2
npm warn     node_modules/@radix-ui/react-popper
npm warn   30 more (@radix-ui/react-arrow, @radix-ui/react-compose-refs, ...)
npm warn
npm warn Could not resolve dependency:
npm warn peer react@"^18.3.1" from react-dom@18.3.1
npm warn node_modules/react-dom
npm warn   peer react-dom@">=16.8.0" from @floating-ui/react-dom@2.1.2
npm warn   node_modules/@floating-ui/react-dom
npm warn   15 more (@radix-ui/react-arrow, ...)
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
```


when installing via `npm install react react-dom @excalidraw/excalidraw` — as described in the README, it seems to install v18 by default and there's no issues.

In another project where we use Excalidraw, where we previously used React 18 and now updated it to React 19, I get 
quite a few peer dependency issue as well e.g `node_modules/@excalidraw/excalidraw/node_modules/@radix-ui/react-tabs`

```
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: @radix-ui/react-tabs@1.0.2
npm warn Found: react@19.0.0
npm warn node_modules/react
npm warn   react@"19.0.0" from the root project
npm warn   34 more (@floating-ui/react-dom, @radix-ui/react-arrow, ...)
npm warn
npm warn Could not resolve dependency:
npm warn peer react@"^16.8 || ^17.0 || ^18.0" from @radix-ui/react-tabs@1.0.2
npm warn node_modules/@excalidraw/excalidraw/node_modules/@radix-ui/react-tabs
npm warn   @radix-ui/react-tabs@"1.0.2" from @excalidraw/excalidraw@0.18.0
npm warn   node_modules/@excalidraw/excalidraw
npm warn
npm warn Conflicting peer dependency: react@18.3.1
npm warn node_modules/react
npm warn   peer react@"^16.8 || ^17.0 || ^18.0" from @radix-ui/react-tabs@1.0.2
npm warn   node_modules/@excalidraw/excalidraw/node_modules/@radix-ui/react-tabs
npm warn     @radix-ui/react-tabs@"1.0.2" from @excalidraw/excalidraw@0.18.0
npm warn     node_modules/@excalidraw/excalidraw
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: @radix-ui/react-tabs@1.0.2
npm warn Found: react-dom@19.0.0
npm warn node_modules/react-dom
npm warn   react-dom@"19.0.0" from the root project
npm warn   10 more (@floating-ui/react-dom, @radix-ui/react-arrow, ...)
npm warn
npm warn Could not resolve dependency:
npm warn peer react-dom@"^16.8 || ^17.0 || ^18.0" from @radix-ui/react-tabs@1.0.2
npm warn node_modules/@excalidraw/excalidraw/node_modules/@radix-ui/react-tabs
npm warn   @radix-ui/react-tabs@"1.0.2" from @excalidraw/excalidraw@0.18.0
npm warn   node_modules/@excalidraw/excalidraw
npm warn
npm warn Conflicting peer dependency: react-dom@18.3.1
npm warn node_modules/react-dom
npm warn   peer react-dom@"^16.8 || ^17.0 || ^18.0" from @radix-ui/react-tabs@1.0.2
npm warn   node_modules/@excalidraw/excalidraw/node_modules/@radix-ui/react-tabs
npm warn     @radix-ui/react-tabs@"1.0.2" from @excalidraw/excalidraw@0.18.0
npm warn     node_modules/@excalidraw/excalidraw
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: @radix-ui/react-context@1.0.0
npm warn Found: react@19.0.0
npm warn node_modules/react
npm warn   react@"19.0.0" from the root project
npm warn   34 more (@floating-ui/react-dom, @radix-ui/react-arrow, ...)
npm warn
npm warn Could not resolve dependency:
npm warn peer react@"^16.8 || ^17.0 || ^18.0" from @radix-ui/react-context@1.0.0
npm warn node_modules/@excalidraw/excalidraw/node_modules/@radix-ui/react-tabs/node_modules/@radix-ui/react-

and so on…
```

--- a/packages/excalidraw/package.json
+++ b/packages/excalidraw/package.json
@@ -76,7 +76,7 @@
     "@excalidraw/mermaid-to-excalidraw": "1.1.2",
     "@excalidraw/random-username": "1.1.0",
     "@radix-ui/react-popover": "1.1.6",
-    "@radix-ui/react-tabs": "1.0.2",
+    "@radix-ui/react-tabs": "1.1.3",
     "browser-fs-access": "0.29.1",
     "canvas-roundrect-polyfill": "0.0.1",
     "clsx": "1.1.1",

